plugins/web/ui/src/app.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<link rel="icon" href="%sveltekit.assets%/vast.svg" />
```

Comment:
I cannot comment on the SVG file directly, but there are two identical vast.svg files in this PR.

Additionally there's a favicon.png file that could probably also be the same vast.svg instead.